packages/stylelint-config-airbnb/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "stylelint-config-airbnb",
+  "version": "0.0.0",
+  "description": "Airbnb's stylelint config, following our styleguide",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "npm run lint"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/airbnb/css.git"
+  },
+  "keywords": [
+    "airbnb",
+    "css",
+    "lint",
+    "stylelint",
+    "stylelint-config",
+    "stylelint-config-airbnb"
+  ],
+  "author": "Stephen Edgar",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/airbnb/css/issues"
+  },
+  "homepage": "https://github.com/airbnb/css#readme",
+  "devDependencies": {
+    "eslint": "^3.0.1",
+    "eslint-config-airbnb-base": "^4.0.0",
+    "eslint-plugin-import": "^1.10.2",
+    "eslint-plugin-jsx-a11y": "^1.5.5",
+    "eslint-plugin-react": "^5.2.2",
+    "stylelint": "^6.9.0",
+    "stylelint-scss": "^1.2.1"
+  },
+  "peerDependencies": {
+    "stylelint": "^6.9.0",
```

Comment:
Done in https://github.com/ntwb/css/commit/43f6dbfdaa3a906df239be6a9d951938d7e8e57d